docs/src/Pages/Components/DateTimePicker/CustomDateTimePicker.example.jsx

```diff
@@ -1,7 +1,9 @@
 import React, { PureComponent, Fragment } from 'react';
 import { DateTimePicker } from 'material-ui-pickers';
-import { IconButton, Icon, InputAdornment } from '@material-ui/core';
-import { withUtilsService } from '../../../_shared/UtilsServiceContext';
+import { IconButton, InputAdornment } from '@material-ui/core';
```

Comment:
Oopsy, path imports please. Or whole mui will appear in bundle

Reply:
🤦‍♂️

Reply:
But this is a docs file! @TrySound I am bundling full material-ui in docs 🤕

Reply:
Okay then